### PR TITLE
[T4A3][T16-B3] Varun Gupta

### PR DESCRIPTION
**Pull request by varung97**
Ready for review

### CODE REVIEW

**Comment by mauris** [comment]
Do take note of the title format change I've made.

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's Contact details in the addressbook. Not meant to be
```
Comment (lohvht-zz) [inline src/seedu/addressbook/data/person/Contact.java:4]: No need to mention where  this class is used

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * used directly, but rather meant to be subclassed.
+ * 
+ * @author varungupta
+ */
+public class Contact {
+	protected String value;
+	protected boolean isPrivate;
+
```
Comment (lohvht-zz) [inline src/seedu/addressbook/data/person/Contact.java:12]: none of the subclasses are using the parent class's constructor, you should create a constructor for the parent class. If you're unsure about inheritance, there are many resources online that talk about this topic, such as this tutorial by Sun Oracle: https://docs.oracle.com/javase/tutorial/java/IandI/subclasses.html

**Comment by lohvht-zz** [comment]
@varung97 added some comments, please code this PR after you're done reading the comments

**Pull request closed without merging**